cmd/datamon/cmd/bundle.go

```diff
@@ -36,7 +36,7 @@ func init() {
 	rootCmd.AddCommand(bundleCmd)
 
 	bundleDescriptorTemplate = func() *template.Template {
-		const listLineTemplateString = `{{.ID}} , {{.Timestamp}} , {{.Message}}`
+		const listLineTemplateString = `{{.ID}} , {{.Timestamp}} , {{.Message}} {{printf "\n"}}`
```

Comment:
```suggestion
		const listLineTemplateString = `{{.ID}} , {{.Timestamp}} , {{.Message}}`
```

Reply:
We need the newline as part of the template.

Reply:
i think [fred is also saying](https://github.com/oneconcern/datamon/pull/375/#discussion_r372628175) we need the newline as part of the output api, just within zap settings instead of a custom `fmt.Printf` call?

Reply:
Ok